less/v2/apps.less

```diff
@@ -664,12 +664,6 @@
   height: 1em;
   display: inline-block;
   background: transparent no-repeat center / contain;
-
-  &.cp { background-image: url("icons/currency/copper.webp"); }
-  &.sp { background-image: url("icons/currency/silver.webp"); }
-  &.ep { background-image: url("icons/currency/electrum.webp"); }
-  &.gp { background-image: url("icons/currency/gold.webp"); }
-  &.pp { background-image: url("icons/currency/platinum.webp"); }
```

Comment:
We use these in some journals, such as in Phandelver & Below, for explaining currencies, so they shouldn't be removed.

Reply:
Added back in.